### PR TITLE
docs(Alert): add readme

### PR DESCRIPTION
**Pull request by yuberdysheva**

### CODE REVIEW

File: src/components/Alert/README.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+<!--/GITHUB_BLOCK-->
+
+## Alert message
+
+`message` - The message of an alert should completely explain the alert's content.
+
+## Alert onClose
+
+`onClose` - enabled close button and defined function in click to this button.
```
Comment (korvin89) [inline src/components/Alert/README.md:167]: Maybe smth like:
`a callback function that is called when the close button of the alert is clicked. When the this property is defined, a close button is being displayed.`

File: src/components/Alert/README.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+  onClose={() => alert('Close button pressed')}
+  title="Alert has close"
+  message="Alert has close"
+/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+### Align
+
```
Comment (korvin89) [inline src/components/Alert/README.md:192]: Maybe add some description? For example
`This property determines how the content inside the Alert component is vertically aligned`

File: src/components/Alert/README.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+`baseline` - align used bu default.
```
Comment (korvin89) [inline src/components/Alert/README.md:193]: `bu` -> `by`

File: src/components/Alert/README.md [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+
+`center` - centered all content in vertical direction, useful if for some reason your actions take more space than text
```
Comment (korvin89) [inline src/components/Alert/README.md:195]: `centered all content in vertical direction` -> `the content is vertically centered within the Alert component`

File: src/components/Alert/README.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+or
+needed icon to be in the middle of the card.
```
Comment (korvin89) [inline src/components/Alert/README.md:197]: There is no need in two lines here

File: src/components/Alert/README.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+<!--GITHUB_BLOCK-->
+
+```tsx
+<Alert align="vertical" title="Vertical" message="Vertical direction" actions={<Button>button</Button>}/>
+<Alert align="horizontal" title="Horizontal" message="Horizontal direction" actions={<Button>button</Button>}/>
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Properties
```
Comment (korvin89) [inline src/components/Alert/README.md:219]: Let's add column `Type` like [here](https://github.com/gravity-ui/uikit/tree/main/src/components/controls/TextInput#properties) and remove bold style from names

File: src/components/Alert/README.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+
+| Name          | Description                                                                  |   Default    |
+| :------------ | :--------------------------------------------------------------------------- | :----------: |
+| **theme**     | Alert appearance: `"normal"` `"info"` `"positive"` `"warning"` `"dangerous"` |  `"normal"`  |
```
Comment (IsaevAlexandr) [inline src/components/Alert/README.md:223]: is it possible for a landing page to set a default prop with a `theme` other than `normal` on for preview? Now the component is indistinguishable from the card on the preview

**Review by IsaevAlexandr — APPROVED** [review]

File: src/components/Alert/README.md (hunk 7 above)
Comment (yuberdysheva) [reply src/components/Alert/README.md:223]: Yes, it is possible, but could it mislead people that this is the default view?
There will be another section with all the themes

File: src/components/Alert/types.ts [hunk 8]
```diff
@@ -23,10 +23,6 @@ export interface AlertProps {
      */
     layout?: 'vertical' | 'horizontal';
     view?: AlertView;
-    /**
-     * Define function and close button will be appear
-     */
-    hasClose?: boolean;
```
[the remark this answers is not in the document]
Comment (yuberdysheva) [reply src/components/Alert/types.ts:29]: I already did it)

File: src/components/Alert/README.md (hunk 6 above)
Comment (yuberdysheva) [reply src/components/Alert/README.md:219]: Ok but be aware that other components have bold and different names of columns. I.e. [Button](https://github.com/gravity-ui/uikit/blob/main/src/components/Button/README.md#properties)

**Review by korvin89 — APPROVED** [review]

**Pull request merged**